mavlink_vehicles/mavlink_vehicles.cc

```diff
@@ -1167,6 +1173,7 @@ void mav_vehicle::update()
 
         // Get back to AUTO mode to continue the mission
         if (this->detour_waypoint_autocontinue) {
+            this->is_our_control = true;
```

Comment:
I agree. But I'm afraid I won't be able to move this commit to another PR without a lot of rebase. Do you think it is really necessary?